### PR TITLE
First release of the Templates package (VAV system)

### PR DESCRIPTION
**Pull request by AntoineGautier**
Draft PR to initiate the review process: please review as it was intended to be merged to MBL master.

The classes of interest for the review are within `Buildings.Templates`.

(If useful I have started putting my notes about template development at https://antoinegautier.github.io/template-dev-docs/.)

TODO

- [x] Pull last commit from `issue1913_g36_final`
- [x] Add release notes
- [x] Debug G36 cases with OCT (OCT-4b0cd2bf7110599ad0f5dab28001f198f5f2c661 solves Modelon issue #2022030739000145 that prevented from compiling the templates after refactoring with a master record and no dependency to ExternData.)
    - This is a bug in OCT tracked by Modelon#2023022839000276 as of 3/7/23.

### CODE REVIEW

**Review by AntoineGautier — COMMENTED** [review]
@JayHuLBL Please see my inline comments. I also pushed some additional modifications in my last commit.

**Comment by JayHuLBL** [comment]
@AntoineGautier I addressed your comments in the last commits. Please check if it is fine.

File: Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Controller.mo [hunk 1]
```diff
@@ -1,25 +1,27 @@
 within Buildings.Controls.OBC.ASHRAE.G36.AHUs.MultiZone.VAV;
 block Controller "Multizone VAV air handling unit controller"
 
-  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard eneStd=Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard.ASHRAE90_1
+  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard eneStd(
+    start=Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard.ASHRAE90_1)
```
Comment (AntoineGautier) [inline Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Controller.mo:5]: Not sure that the `start` attribute assignment is needed here as the parameter is always enabled.

File: Buildings/Controls/OBC/ASHRAE/G36/Types/OutdoorAirSection.mo [hunk 2]
```diff
@@ -1,8 +1,7 @@
 within Buildings.Controls.OBC.ASHRAE.G36.Types;
 type OutdoorAirSection = enumeration(
-    DedicatedDampersAirflow   "Seperate dampers for ventilation and econmomizer, with airflow messurement station",
-    DedicatedDampersPressure   "Seperate dampers for ventilation and econmomizer, with differential pressure sensor",
-    NoEconomizer   "No economizer",
+    DedicatedDampersAirflow   "Separate dampers for ventilation and economizer, with airflow measurement station",
+    DedicatedDampersPressure   "Separate dampers for ventilation and economizer, with differential pressure sensor",
     SingleDamper   "Single damper for ventilation and economizer, with airflow messurement station")
```
Comment (AntoineGautier) [inline Buildings/Controls/OBC/ASHRAE/G36/Types/OutdoorAirSection.mo:5]: Typo: messurement

File: Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Controller.mo [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
     "Energy standard, ASHRAE 90.1 or Title 24";
-  parameter Types.VentilationStandard venStd=Buildings.Controls.OBC.ASHRAE.G36.Types.VentilationStandard.ASHRAE62_1
+  parameter Types.VentilationStandard venStd(
+    start=Buildings.Controls.OBC.ASHRAE.G36.Types.VentilationStandard.ASHRAE62_1)
```
Comment (AntoineGautier) [inline Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Controller.mo:8]: Same comment as above.

**Review by AntoineGautier — COMMENTED** [review]
@JayHuLBL Just a few remaining comments.

**Comment by AntoineGautier** [comment]
@akapoor66 The commit 4b5b246 includes the additional annotations needed to disable advanced modeling parameters that should not be displayed in the config page.

**Comment by mwetter** [comment]
@AntoineGautier : Is this ready to be completed and reviewed so that we can reduce the PR backlog, and have this tested in other tools CI testing which only works from our master branch?

**Comment by AntoineGautier** [comment]
@mwetter @JayHuLBL This is ready for a first review.

**Pull request merged**